examples/threadExample/src/main/res/layout/fragment_async_query.xml

```diff
@@ -0,0 +1,43 @@
+<!--
```

Comment:
other layout use `~` as well, besides `threadExample` is the only sample using a header for layouts, other samples don't, thinking about removing it all together